templates/application-qr-code-generator.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: glueops-pull-request-bot
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+spec:
+  destination:
+    name: "in-cluster"
+    namespace: glueops-core-pull-request-bot
+  project: glueops-core
+  syncPolicy:
+    syncOptions:
+      - Replace=true
+      - CreateNamespace=true
+    automated:
+      prune: true
+      selfHeal: true
+    retry:
+      backoff:
+        duration: 5s
+        maxDuration: 3m0s
+        factor: 2
+      limit: 2
+    source:
+        repoURL: 'https://helm.gpkg.io/project-template'
+        targetRevision: 0.2.0
+        helm:
+            valueFiles:
+            - values.yaml
+            values: |-
+            replicaCount: '2'
+            image: 
```

Comment:
can you confirm what the current pod requests are? how many resources are being allocated? right now? you can find this in grafana.<captain-domain>